app/web/src/features/communities/events/EventsTab.tsx

```diff
@@ -0,0 +1,97 @@
+import { Typography } from "@material-ui/core";
+import classNames from "classnames";
+import Alert from "components/Alert";
+import Button from "components/Button";
+import CircularProgress from "components/CircularProgress";
+import TextBody from "components/TextBody";
+import { useHistory } from "react-router-dom";
+import { newEventRoute } from "routes";
+import hasAtLeastOnePage from "utils/hasAtLeastOnePage";
+import makeStyles from "utils/makeStyles";
+
+import { useCommunityPageStyles } from "../CommunityPage";
+import { EVENTS_EMPTY_STATE } from "../constants";
+import { CREATE_AN_EVENT, SEE_MORE_EVENTS_LABEL } from "./constants";
+import EventCard from "./EventCard";
+import { useListAllEvents } from "./hooks";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    display: "grid",
+    rowGap: theme.spacing(2),
+    paddingBlockStart: theme.spacing(1),
+    paddingBlockEnd: theme.spacing(5),
+    justifyItems: "start",
+  },
+  container: {
+    [theme.breakpoints.down("xs")]: {
+      gridTemplateColumns: "1fr",
+      gridGap: theme.spacing(2),
+    },
+    display: "grid",
+  },
+  eventsCard: {
+    width: "100%",
+  },
+  moreEventButton: {
+    justifySelf: "center",
+  },
+}));
+
+interface EventsTabProps {
+  pastEvents?: boolean;
+  tabTitle: string;
+}
+
+export default function EventsTab({
+  pastEvents = false,
+  tabTitle,
+}: EventsTabProps) {
+  const classes = { ...useCommunityPageStyles(), ...useStyles() };
+  const history = useHistory();
+
+  const { data, error, hasNextPage, fetchNextPage, isLoading } =
+    useListAllEvents({ pastEvents });
+
+  return (
+    <div className={classes.root}>
+      {!pastEvents && (
+        <Button onClick={() => history.push(newEventRoute)}>
+          {CREATE_AN_EVENT}
+        </Button>
+      )}
+      <Typography variant="h2">{tabTitle}</Typography>
+      {error && <Alert severity="error">{error.message}</Alert>}
```

Comment:
IMO it makes more sense to have title - error - create button

Reply:
Hmm yeah that'd also be more consistent with the other pages 👍